app/dashboard/models.py

```diff
@@ -783,6 +784,46 @@ def is_project_type_fulfilled(self):
             fulfilled = self.interested.filter(pending=False).exists()
         return fulfilled
 
+    @property
+    def tips(self):
+        try:
```

Comment:
Think we could add docstrings to the new methods?

Reply:
done